File: app/src/main/java/github/hellocsl/cursorwheellayout/widget/SimpleTextCursorWheelLayout.java
package github.hellocsl.cursorwheellayout.widget;

import android.content.Context;
import android.util.AttributeSet;
import android.view.View;

import github.hellocsl.cursorwheel.CursorWheelLayout;
import github.hellocsl.cursorwheellayout.R;

/**
 *
 * Created by chensuilun on 16-4-19.
 */
public class SimpleTextCursorWheelLayout extends CursorWheelLayout {
    public static final int MENU_COUNT = 10;
    public static final int INDEX_SPEC = 9;

    public SimpleTextCursorWheelLayout(Context context, AttributeSet attrs) {
        super(context, attrs);
    }


    @Override
    protected void onInnerItemSelected(View v) {
        super.onInnerItemSelected(v);
        if (v == null) {
            return;
        }
        View tv = v.findViewById(R.id.wheel_menu_item_tv);
        tv.animate().scaleX(2).scaleY(2);
    }


    @Override
    protected void onInnerItemUnselected(View v) {
        super.onInnerItemUnselected(v);
        if (v == null) {
            return;
        }
        View tv = v.findViewById(R.id.wheel_menu_item_tv);
        tv.animate().scaleX(1).scaleY(1);
    }


}

File: app/src/main/java/github/hellocsl/cursorwheellayout/widget/SimpleTextCursorWheelLayout.kt
package github.hellocsl.cursorwheellayout.widget

import android.content.Context
import android.util.AttributeSet
import android.view.View
import github.hellocsl.cursorwheel.CursorWheelLayout
import github.hellocsl.cursorwheellayout.R

/**
 *
 * Created by chensuilun on 16-4-19.
 */
class SimpleTextCursorWheelLayout(context: Context, attrs: AttributeSet?) : CursorWheelLayout(context, attrs) {
    override fun onInnerItemSelected(v: View?) {
        super.onInnerItemSelected(v)
        val tv = v?.findViewById<View>(R.id.wheel_menu_item_tv)
        tv?.animate()?.scaleX(2f)?.scaleY(2f)
    }

    override fun onInnerItemUnselected(v: View?) {
        super.onInnerItemUnselected(v)
        val tv = v?.findViewById<View>(R.id.wheel_menu_item_tv)
        tv?.animate()?.scaleX(1f)?.scaleY(1f)
    }

    companion object {
        const val MENU_COUNT = 10
        const val INDEX_SPEC = 9
    }
}